src/app/utils/warranty.ts

```diff
@@ -0,0 +1,106 @@
+import { warranty } from 'app/constants/warranty';
+
+// 必要書類
+interface requiredDocumentCheckList {
+  report: boolean,
```

Comment:
```suggestion
  report: boolean;
```
以下同様です。
このあたりは、pre commit hookでlintかける設定を入れておけば防げると思います。
少し手が空いた時にでも、化研のリポジトリで追加しているので、参考にしてみてください。